### PR TITLE
Blog ehcache3 jsr107 spring

### PR DESCRIPTION
**Pull request by gibsong**

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Authors
+ Authors
```
Comment (ljacomet) [inline _data/authors.yml]: Why uncomment this? pretty sure that's not expected content for that yml file

File: _posts/blog/2016-05-12-ehcache3_jsr107_spring.adoc [hunk 2]
```diff
@@ -0,0 +1,276 @@
+---
+layout: post
+title: "Ehcache 3, JSR-107 and Spring 4.2"
+headline: "Ehcache 3 has been released!"
```
Comment (ljacomet) [inline _posts/blog/2016-05-12-ehcache3_jsr107_spring.adoc]: I believe the headline should be a longer description of what your post is about.

File: _data/authors.yml (hunk 1 above)
Comment (gibsong) [inline _data/authors.yml]: That was a mistake.  I added the comment back in.

File: _posts/blog/2016-05-12-ehcache3_jsr107_spring.adoc (hunk 2 above)
Comment (gibsong) [inline _posts/blog/2016-05-12-ehcache3_jsr107_spring.adoc]: I updated the the headline.  I hope it sounds good.  If not let me know and I will think of something else.

**Comment by ljacomet** [comment]
Closing as I merged command line

**Pull request closed without merging**